infohub2024/routes/web.php

```diff
@@ -88,10 +88,11 @@
         Route::get('/edit/{menuItem}', [MenuController::class, 'edit'])->name('edit');
         Route::post('/toggle-status/{menuItem}', [MenuController::class, 'toggleStatus'])->name('toggleStatus');
 
-        Route::post('/menu-items', [MenuItemController::class, 'store'])->name('menu-items.store');
+//        Route::post('/menu-items', [MenuItemController::class, 'store'])->name('menu-items.store');
         Route::get('/get-menu-items', [MenuController::class, 'getMenuItems']);
         Route::get('/get-menu-items-with-files', [MenuController::class, 'getMenuItemsWithFiles']);
 
```

Comment:
You have to group  the routes accordingly to the functionality of the routes it performs.